Comparator.dcm

```diff
@@ -61,7 +61,7 @@ F https://www.st.com/resource/en/datasheet/lm139.pdf
 $ENDCMP
 #
 $CMP LM393
-D Low-Power, Low-Offset Voltage, Dual Comparators, DIP-8/SOIC-8/SSOP-8
+D Low-Power, Low-Offset Voltage, Dual Comparators, DIP-8/SOIC-8/TO-99-8
```

Comment:
There is a difference between `LM393` and `LM393-N`
* http://www.ti.com/lit/ds/symlink/lm393-n.pdf
* http://www.ti.com/lit/ds/symlink/lm393.pdf

You did link to the `-N` datasheet. 
To Device in TO-99-8 package is called LM293-N
The datasheet for the non `-N` does not have a TO package.

Reply:
Fixed